pymisp/api.py

```diff
@@ -140,11 +135,7 @@ def get_live_query_acl(self):
 
     @deprecated(reason="Use ExpandedPyMISP.describe_types_local", version='2.4.110', action='default')
     def get_local_describe_types(self):
-        with open(os.path.join(self.resources_path, 'describeTypes.json'), 'rb') as f:
-            if OLD_PY3:
-                describe_types = json.loads(f.read().decode())
-            else:
-                describe_types = json.load(f)
+        describe_types = self._load_json(os.path.join(self.resources_path, 'describeTypes.json'))
```

Comment:
I think this could also be changed to use the descripe_types variable from abstract.py by importing the loaded content.

Reply:
:+1: